projects/storefrontlib/src/cms-components/navigation/search-box/search-box.component.html

```diff
@@ -48,7 +48,7 @@
           [routerLink]="
             {
               cxRoute: 'product',
-              params: suggestion | stripHtml
+              params: suggestion
```

Comment:
In line 61 please use new `nameHtml` property. 
Btw. you can remove line 62, because it's replaced in runtime by the `[innerHtml]` directive